pdfixed/include/bm/pdfixed/pd_common.h

```diff
@@ -73,7 +73,7 @@ typedef struct p4_pd_packets_meter_spec {
 
 typedef struct p4_pd_bytes_meter_spec {
   uint32_t cir_kbps;
-  uint32_t cburst_kbits;
```

Comment:
we are not changing the interface, unless there is a bug that needs to be fixed

Reply:
There is no bug in the code.  However, one reviewer has already made a comment saying "burst sizes are specified in multiples of 1000 bits (Kbits would be confusing to many people, I think, as it would often be interpreted as multiples of 1024 bits)."  To address the comment, I decided to use kilo octets for burst_size.  On second thought, I think "kilo" is still confusing because folks will ask if it is 1024 or 1000?  

Why not use octets for burst_size? 

A size in several specifications uses octets.  See RFC 8200 for IPv6 Payload length which uses octets.

Reply:
If the code comments and conversions are correct, it is not kilo octets.  It is 10^3 _bits_.  Or you could say it is in multiples of 125 bytes/octets.

Reply:
i know the code conversion is correct.  But I changed the code to use kilo octets.  But kilo is ambiguous.  So, we could use `10^3 octets`.

Reply:
Or `10^3 bytes` would work too.  Then the original code conversion remains unchanged.